changelogs/fragments/6785-openbsd_pkg_pkg_info_handling.yml

```diff
@@ -0,0 +1,2 @@
+xes:
```

Comment:
```suggestion
bugfixes:
```

Reply:
Thanks, no commit before the first coffee ;)